API.md

```diff
@@ -31,4 +31,4 @@ Parameter name                | Value   | Description
 api_token                     | string  | A token that identifies a unique user
 prediction[description]       | string  | Your prediction statement
 prediction[deadline]          | date    | When it will/won't have happened
-prediction[private]           | boolean | True if you want it to be private (optional)
+prediction[visibility]        | string  | visible_to_everyone (default), visible_to_creator
```

Comment:
This looks like a breaking change to the API. I'm note sure if anyone is currently using the API (server logs will confirm) but if they are we probably should introduce a new version of the API or remain backwards compatible.